facia-press/app/AppLoader.scala

```diff
@@ -19,25 +20,18 @@ class AppLoader extends FrontendApplicationLoader {
   override def buildComponents(context: Context): FrontendComponents = new BuiltInComponentsFromContext(context) with AppComponents
 }
 
-trait Controllers {
-  def wsClient: WSClient
-  def toolPressQueueWorker: ToolPressQueueWorker
-  def liveFapiFrontPress: LiveFapiFrontPress
-  def draftFapiFrontPress: DraftFapiFrontPress
-  lazy val healthCheck = wire[HealthCheck]
-  lazy val applicationController: Application = wire[Application]
-}
+trait AppComponents extends FrontendComponents {
```

Comment:
I think you're right, it does make it easier to digest. Let's see how it scales!